vince.json

```diff
@@ -0,0 +1,28 @@
+{
+  "response": [
+    {
+        "maintainer": "Rahul K (rk134)",
+        "oem": "xiaomi",
+        "device": "Redmi 5 Plus",
+        "filename": "crDroidAndroid-12.0-20220216-vince-v8.2.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/vince/8.x/crDroidAndroid-12.0-20220216-vince-v8.2.zip/download",
+        "timestamp": 1645013741,
+        "md5": "3d55ea94333966f4253e434d37dec8b8",
+        "sha256": "d492e1816094dcc287c83d9fcdae8b7fd5b155410417f395d6ebf845afa411ca",
+        "size": 738626479,
+        "version": "8.2",
+        "buildtype": "monthly",
```

Comment:
Beta due to no force encrypt
https://github.com/crdroidandroid/android_device_xiaomi_vince/blob/12.0/rootdir/etc/fstab.qcom#L11#L12

Reply:
ok!